lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1708,3 +1709,60 @@ def test_download_csv(self):
     def get_url(course_id):
         # This strategy allows us to test with invalid course IDs
         return reverse('team_membership_bulk_management', args=[course_id])
+
+    def test_upload_valid_csv_simple(self):
+        self.create_and_enroll_student(username='a_user')
+        csv_content = 'user,mode,topic_0' + '\n'
+        csv_content += 'a_user, masters, team wind power'
+        csv_file = SimpleUploadedFile('test_file.csv', csv_content.encode('utf8'), content_type='text/csv')
+        self.client.login(username=self.users['course_staff'].username, password=self.users['course_staff'].password)
+        self.make_call(reverse('team_membership_bulk_management', args=[self.good_course_id]),
+                       201, method='post',
+                       data={'csv': csv_file}, user='staff'
+                       )
+
+    def test_upload_invalid_teamset(self):
+        self.create_and_enroll_student(username='a_user')
```

Comment:
There seems like a lot of repeated test setup here. Is there any way to reduce this a bit?

Reply:
I would appreciate any suggestions. While the setup appears repeated, at least with regards to 'a_user'. There are multiple variations of different scenarios that I don't think is worthwhile to look for a way to refactor.